src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -91,7 +91,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('if_match')
         c.argument('if_none_match')
 
-    for item in ['delete', 'show', 'update', 'show-connection-string', 'keys', 'network-rule']:
+    for item in ['delete', 'show', 'update', 'show-connection-string', 'keys', 'network-rule', 'revoke-delegation-keys']:  # pylint: disable=line-too-long
```

Comment:
why disabled line too long here?

Reply:
Otherwise, `azdev style` will cause error and I cannot pass the CI